src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/StackGuard.cs

```diff
@@ -67,17 +67,15 @@ private R RunOnEmptyStackCore<R>(Func<object, R> action, object state)
                 // Using default scheduler rather than picking up the current scheduler.
                 Task<R> task = Task.Factory.StartNew(action!, state, CancellationToken.None, TaskCreationOptions.DenyChildAttach, TaskScheduler.Default);
 
-                TaskAwaiter<R> awaiter = task.GetAwaiter();
```

Comment:
(nit) thoughts on consolidating these two, mostly duplicated, files?

Reply:
I think we should, but I'd like to do that separately.  The Expressions one has a bunch of additional stuff in it.